### PR TITLE
Add documentUrl for attached docs in EntryPage

### PR DESCRIPTION
**Pull request by priyesh777**
Related to  https://github.com/idmc-labs/Helix2.0/issues/130
Related to https://github.com/idmc-labs/Helix2.0/issues/136
Related to https://github.com/idmc-labs/Helix2.0/issues/143
Related to https://github.com/idmc-labs/Helix2.0/issues/68
Related to https://github.com/idmc-labs/Helix2.0/issues/132
Related to https://github.com/idmc-labs/Helix2.0/issues/135

Depends on https://github.com/idmc-labs/helix-server/pull/190

Addresses 
- DocumentUrl added to Entry Detail
- Age changed to "Age/Sex"
- Removed disaggregation by male or female
- Changed Traffic-light input names

## This PR doesn't introduce any:

- [x] temporary files, auto-generated files or secret keys
- [x] build works
- [x] es-lint issues
- [x] typescript issues
- [x] `console.log` meant for debugging
- [x] typos
- [x] unwanted comments

## This PR contains valid:

- [x] permission checks
- [x] translations

### CODE REVIEW

File: src/views/Entry/EntryForm/FigureInput/index.tsx [hunk 1]
```diff
@@ -774,7 +740,7 @@ function FigureInput(props: FigureInputProps) {
                     <div className={styles.block}>
                         <Header
                             size="extraSmall"
-                            heading="Age"
+                            heading="Age/Sex"
```
Comment (tnagorra) [inline src/views/Entry/EntryForm/FigureInput/index.tsx:743]:
```suggestion
                            heading="Age / Sex"
```

File: src/views/Entry/EntryForm/index.tsx [hunk 2]
```diff
@@ -676,6 +677,18 @@ function EntryForm(props: EntryFormProps) {
...
+            onValueSet((oldValue) => ({
+                ...oldValue,
+                details: {
+                    ...oldValue.details,
+                    documentUrl: undefined,
+                },
+            }));
+        }, [onValueSet],
+    );
+
```
Comment (tnagorra) [inline src/views/Entry/EntryForm/index.tsx:691]: We don't need to handle document url differently

File: src/views/Entry/EntryForm/FigureInput/index.tsx [hunk 3]
```diff
@@ -651,40 +651,6 @@ function FigureInput(props: FigureInputProps) {
                             )}
                         />
                     </Row>
-                    <Row>
```
Comment (tnagorra) [inline src/views/Entry/EntryForm/FigureInput/index.tsx:654]: We should also remove these from the schema

**Review by tnagorra — CHANGES_REQUESTED** [review]

File: src/views/Entry/EntryForm/index.tsx (hunk 2 above)
Comment (priyesh777) [reply src/views/Entry/EntryForm/index.tsx:691]: This was actually meant to clear the documentUrl field

Comment (tnagorra) [reply src/views/Entry/EntryForm/index.tsx:692]: Yes, we don't want that!

**Pull request merged**